### PR TITLE
Add an option to define executable name on Windows

### PR DESCRIPTION
**Pull request by malys**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Why? Always include a pull request description ;)

**Comment by malys** [comment]
By default, you consider that the executable on Windows is "compass.bat". However, It isn't always the case. This can be "compass.bat","compass.cmd" or "compass.exe"
With this option, the developer can override the executable name.

an efficient title means a lot. Moreover, I'm not really a little Shakespeare. :).

**Comment by sindresorhus** [comment]
"Why" is always more important than "what" ;)

**Comment by sindresorhus** [comment]
Why isn't the bin extension consistent? Do you have any proof of this? If true I would prefer an automated solution.

**Comment by malys** [comment]
If you install ruby + compass, you have to launch compass.bat.
But on Windows, you can use a portable version of compass to embed ruby and compass in one exe file. See [Portable Compass](https://github.com/jbowtie/compass-windows-exe).

Consequently, the application setup is more simple and self-sufficient.

**Comment by sindresorhus** [comment]
Ok, but I'd prefer to auto-detect it somehow. I don't want users to have to manually specify this. Can probably use https://github.com/isaacs/node-which for that.

File: tasks/lib/compass.js [hunk 1]
```diff
@@ -156,7 +156,14 @@ exports.init = function (grunt) {
     var basePath = options.basePath;
 
     if (process.platform === 'win32') {
-      args.unshift('compass.bat');
+	  if(options.exe){
```
Comment (vladikoff) [inline tasks/lib/compass.js]: strange indent

**Pull request closed without merging**